consensus/XDPoS/utils/types.go

```diff
@@ -168,7 +172,7 @@ func TimeoutSigHash(m *TimeoutForSign) common.Hash {
 
 func (m *Vote) PoolKey() string {
 	// return the voted block hash
-	return m.ProposedBlockInfo.Hash.Hex()
+	return fmt.Sprint(m.ProposedBlockInfo.Round, ":", m.ProposedBlockInfo.Number, ":", m.ProposedBlockInfo.Hash.Hex())
```

Comment:
Please also add m.GapNumber in my recent PR

Reply:
I think we need to discuss the pool key a bit.

Reply:
👍